test/run/flow.js

```diff
@@ -0,0 +1,65 @@
+const {
+  tmpfile,
+  run,
+  tap,
+  t,
+} = require('./')
+
+
+t.test('flow', t => {
+  const ok = tmpfile(t, 'flow/ok.js', `
+    // @flow
+    const t = require(${tap})
+    function square(n: number): number {
+      return n * n;
+    }
+    t.pass('this is fine')
+  `)
+
+  const args = [ok, '--flow']
+
+  run(args, {}, (er, o, e) => {
+    t.equal(er, null)
+    t.matchSnapshot(o)
+    t.end()
+  })
+})
+
+t.test('flow manually', t => {
+  const ok = tmpfile(t, 'flow/ok2.js', `
+    // @flow
+    const t = require(${tap})
+    function square(n: number): number {
+      return n * n;
+    }
+    t.pass('this is fine')
+  `)
+
+  const args = [ok, '--node-arg=--require', '--node-arg=flow-remove-types/register']
+  
+  run(args, {}, (er, o, e) => {
+    t.equal(er, null)
+    t.matchSnapshot(o)
+    t.end()
+  })
+})
+
+
+// t.test('flow manually', t => {
```

Comment:
Remove rather than commenting out.  (We can always pull out of git if it turns out to be necessary, but it usually doesn't.)

Reply:
Sorry about that. Sloppy.